stdlib/Markdown/src/Markdown.jl

```diff
@@ -40,6 +44,18 @@ function docexpr(source::LineNumberNode, mod::Module, s, flavor = :julia)
     :($doc_str($(mdexpr(s, flavor)), $(QuoteNode(source)), $mod))
 end
 
+"""
+    macro md_str(s)
```

Comment:
```suggestion
    @md_str -> Markdown.MD
```

to match the convention used here https://docs.julialang.org/en/v1/base/strings/#Base.Docs.@html_str

Reply:
I've copied the convention I found in [`Artifacts.@artifact_str`](https://docs.julialang.org/en/v1/stdlib/Artifacts/#Artifacts.@artifact_str). If I find the time, I'll try to find all (string literal) macro docs, check for their conventions and file an issue if I find too many divergent styles.